kie-server-parent/kie-server-services/kie-server-services-common/src/main/java/org/kie/server/services/impl/ContainerManager.java

```diff
@@ -30,6 +30,10 @@
     private static final Logger logger = LoggerFactory.getLogger(ContainerManager.class);
 
     public void installContainers(KieServerImpl kieServer, Set<KieContainerResource> containers, KieServerState currentState, KieServerSetup kieServerSetup) {
+        installContainersSync(kieServer, containers, currentState, kieServerSetup);
+    }
+
+    public void installContainersSync(KieServerImpl kieServer, Set<KieContainerResource> containers, KieServerState currentState, KieServerSetup kieServerSetup) {
```

Comment:
installContainersSync seems to do same thing as installContainers. Is there any reason to have both methods here?